src/main/java/org/breedinginsight/brapi/v2/ExperimentController.java

```diff
@@ -61,4 +62,25 @@ public HttpResponse<Response<DataResponse<List<BrAPITrial>>>> getExperiments(
             return HttpResponse.status(HttpStatus.NOT_FOUND, e.getMessage());
         }
     }
+
+    @Get("/${micronaut.bi.api.version}/programs/{programId}" + BrapiVersion.BRAPI_V2 + "/trials/{trialId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ProgramSecured(roleGroups = {ProgramSecuredRoleGroup.ALL})
+    public HttpResponse<Response<HashMap<String, Object>>> getExperimentById(
```

Comment:
I think this return type should be `HttpResponse<Response<BrAPITrial>>`

Reply:
done